tsfc/spectral.py

```diff
@@ -44,14 +32,86 @@ def Integrals(expressions, quadrature_multiindex, argument_multiindices, paramet
 
     :returns: list of integral representations
     """
+    # Rewrite: a / b => a * (1 / b)
+    expressions = replace_division(expressions)
```

Comment:
FWIW, clang won't generate FMAs unless you say either `-ffp-contract=fast` or `-ffast-math`.